### ISSUE TITLE
IF_LOOP True False branch is wrong

### ISSUE THREAD

**Issue by PriyankaBose** [crytic/slither#518]
```
pragma solidity ^0.4.24;

contract PoC {
    
    function withdrawBalance(uint256 element, uint amount) public{
      uint kittycount = 9;
      uint count = 0;
      
      for (uint i=0;i<kittycount;i++) {
        count = count + 1;
      }
    }    
}
```

For the above contract, `node.sons[0]` (which is supposedly the true branch) of the node `IF_LOOP i < kittycount` is coming as `END_LOOP None` and `node.sons[1]` (which is supposedly the false branch) is coming as `EXPRESSION count = count + 1`. However, it should be completely opposite I believe.

**Comment by montyly** [comment]
Hi @PriyankaBose, thank you for reporting this.

We will fix it as soon as possible

**Comment by montyly** [comment]
Hi @PriyankaBose, we fixed this issue in #524, the fix is available in master.

Thank you for reporting this issue.
